src/SceneKit/Defs.cs

```diff
@@ -18,15 +18,17 @@
 namespace XamCore.SceneKit {
 
 	[Watch (3,0)]
-	[Availability (Platform.Mac_10_8 | Platform.iOS_8_0)]
+	[Availability (Platform.Mac_10_8)]
+	[Availability (Platform.iOS_8_0)]
```

Comment:
^ should not those be converted ?

Reply:
That's a good question. Let me take a look...

Reply:
Oh, I looked for ```Availability (Introduced =``` but not the common case, and only in the original patch.